cranelift/codegen/src/ir/exception_table.rs

```diff
@@ -41,10 +54,14 @@ pub struct ExceptionTableData {
     /// branch targets for any branch instruction. The last BlockCall
     /// is the normal-return destination, and the rest correspond to
     /// the tags in `tags` below. Thus, we have the invariant that
-    /// `targets.len() == tags.len() + 1`.
+    /// `targets.len() == items.len() + 1`.
```

Comment:
This length invariant no longer holds because context entries do not have an associated target, right?

Reply:
Updated, thanks!